controllers/common/create_tree.go

```diff
@@ -54,12 +53,14 @@ func CreateTrillianTree(ctx context.Context, displayName string, trillianURL str
 	adminClient := trillian.NewTrillianAdminClient(conn)
 	logClient := trillian.NewTrillianLogClient(conn)
 
-	tree, err := client.CreateAndInitTree(ctx, req, adminClient, logClient)
-
+	timeout := time.Duration(120 * time.Second)
+	ctx2, cancel := context.WithTimeout(ctx, timeout)
```

Comment:
```suggestion
	ctx2, cancel := context.WithTimeout(ctx, timeout)
	defer cancel()
```
https://pkg.go.dev/context#WithTimeout